src/lsm/superblock_free_set.zig

```diff
@@ -72,17 +72,30 @@ pub const SuperBlockFreeSet = struct {
 
     /// Returns the number of released blocks.
     /// Excludes blocks staged to be released.
-    pub fn count_released(set: *SuperBlockFreeSet) u64 {
+    pub fn count_released(set: SuperBlockFreeSet) u64 {
         return set.blocks.count();
     }
 
     /// Returns the number of acquired blocks.
     /// Includes blocks staged to be released.
-    pub fn count_acquired(set: *SuperBlockFreeSet) u64 {
+    pub fn count_acquired(set: SuperBlockFreeSet) u64 {
         return set.blocks.capacity() - set.blocks.count();
     }
 
-    // TODO Add unit tests for count_released() and count_acquired().
+    /// Returns the address of the highest allocated block.
+    pub fn last_acquired(set: SuperBlockFreeSet) ?u64 {
+        var it = set.blocks.iterator(.{
+            .kind = .unset,
+            .direction = .reverse,
+        });
+
+        if (it.next()) |block| {
+            const address = block + 1;
+            return address;
+        } else {
+            return null;
```

Comment:
We could assert before returning null that there are zero acquired blocks in the set.